### PR TITLE
NAP-192 Passenger transportation form ui tuning. Header ui tuning.

### PR DESCRIPTION
**Pull request by rckopteri**
Added select-field to main header and named it as user-menu.
Slightly changed the way how our main buttons are styled.
Changed slightly map placement on the passenger transportation form.

### CODE REVIEW

File: ote/src/cljs/ote/style/base.cljs [hunk 1]
```diff
@@ -13,8 +13,27 @@
...
+(def action-button {:padding-left "1.1em"
+                    :padding-right "1.1em"
+                    :text-transform "uppercase"
+                    :color "#FFFFFF"
+                    :background-color "#1565C0"
+                    :font-size "12px"
+                    :font-weight "bold"
+                    })
 
+(def cancel-button {:padding-left "1.1em"
```
Comment (tatut) [inline ote/src/cljs/ote/style/base.cljs]: move common style defs to `base-button` or something like that, and use it with `merge` for action and cancel buttons

File: ote/src/cljs/ote/views/main.cljs [hunk 2]
```diff
@@ -29,25 +29,50 @@
...
     [ui/menu-item {:primary-text " Näytä debug state"
+                   :selected-text-color (color :grey900)
                    :on-click #(e! (fp-controller/->ToggleDebugState))} ]
     [ui/menu-item {:primary-text "Kirjaudu ulos"
+                   :selected-text-color (color :grey900)
                    :on-click #(e! (fp-controller/->ChangePage :front-page))} ]
     ]
   )
 
+(defn- create-top-nav
```
Comment (tatut) [inline ote/src/cljs/ote/views/main.cljs]: this fn returns the top-nav component and should be named `top-nav` and called as a component `[top-nav e! app]`

File: ote/src/cljs/ote/views/main.cljs [hunk 3]
```diff
@@ -29,25 +29,50 @@
...
+  [:div {:class "topnav"}
+   [:a.main-icon {:href "#home"} [:img {:src "img/icons/nap-logo.svg" }]]
+   [:a.ote-nav { :href "/index.html" } (tr [:common-texts :header-front-page]) ]
+   [:a.ote-nav
+    {:class (is-topnav-active :front-page (:page app))
+     :href "#service-operator"
+     :on-click #(e! (fp-controller/->ChangePage :front-page))
+     } (tr [:common-texts :header-own-service-list]) ]
+   [:div.user-menu {:class (is-user-menu-active app) }
+    (reagent/as-element (user-menu e! (get-in app [:user :name])))
```
Comment (tatut) [inline ote/src/cljs/ote/views/main.cljs]: closing parens

File: ote/src/cljs/ote/style/base.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/style/base.cljs]: I made only one button style, base-button.

File: ote/src/cljs/ote/views/main.cljs (hunk 2 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/main.cljs]: Fixed

File: ote/src/cljs/ote/views/main.cljs (hunk 3 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/main.cljs]: fixed

**Pull request merged**